client-v2/src/components/Clipboard.tsx

```diff
@@ -84,89 +98,99 @@ class Clipboard extends React.Component<ClipboardProps> {
 
   public render() {
     return (
-      <StyledDragIntentContainer
-        active={!this.props.isClipboardOpen}
-        onDragIntentStart={() => this.setState({ preActive: true })}
-        onDragIntentEnd={() => this.setState({ preActive: false })}
-        onIntentConfirm={() => this.props.toggleClipboard(true)}
+      <ClipboardWrapper
+        tabIndex={0}
+        onFocus={this.handleFocus}
+        onBlur={this.handleBlur}
```

Comment:
There's a question here as to how we keep the application focus state sync'ed with the DOM. We've opted for handling focus and blur events explicitly on the relevant components.